daphne/__init__.py

```diff
@@ -1,6 +1,6 @@
 import sys
 
-__version__ = "3.0.2"
+__version__ = "3.0.3"
```

Comment:
Please drop this. It's handled on release.

Reply:
Fixed